src/containers/TemplateBuilder/evaluatorGui/typeHelpers.ts

```diff
@@ -149,9 +149,11 @@ export const convertTypedEvaluationToBaseType: ConvertTypedEvaluationToBaseType
       }
       const operatorResult: {
         operator: Operator
+        type?: string
         children: EvaluatorNode[]
       } = {
         operator: evaluation.asOperator.operator,
+        type: evaluation.asOperator.type,
```

Comment:
So does this pass `type: undefined` if type not specified? Does that work?

Reply:
it seems to work, if you toggle it while switching to show as gui you'll see it disappearing